### DISCUSSION TITLE
combining more than two features

### DISCUSSION THREAD

**Discussion by arashtayyebi** [rdkit/rdkit#5513] [General]
Hi all,
I have some important features like as ( '[O]', 'CN(C)(C)C', ' [CH](-[CH])(-[O])-[O]', '[c](:[c])(:[c])-[OH]', '[c](:[cH])(:[c])-[S]', ' [CH2](-[CH2])-[NH]') and I want to combine them and make a new SMILES. 
Is there any way to see all the possibilities to integrate these fragments?

**Comment by greglandrum** [comment]
@arashtayyebi can you provide more information about what you mean by integrating or combining fragments?

**Reply by arashtayyebi** [reply]
@greglandrum The idea is like as combining two molecules
mol1 = Chem.MolFromSmiles('[NH](-[CH2]-[CH2])-[C](-[c])=[O]',sanitize=False)
mol2 = Chem.MolFromSmiles('[CH2](-[CH])-[CH2]')
if I try :
combo1 = Chem.CombineMols(mol1,mol2)
it works,
but if I have more than two molecules, how can I combine them like as combo1?
for instance, if I input mol3:
mol3 = Chem.MolFromSmiles('[O]=[CH]')
so the following code will not work
combo2 = Chem.CombineMols(mol1,mol2,mol3)
